repositories.tf

```
@@ -0,0 +1,105 @@
+locals {
+  default_repo = {
+    branch_protection = {
+      required_approving_review_count = 2
```

Remove the alternative in `core-cloud` and have 1 as sensible default.

```suggestion
      required_approving_review_count = 1
```